extensions/sql-migration/src/models/stateMachine.ts

```diff
@@ -173,11 +177,13 @@ export class MigrationStateModel implements Model, vscode.Disposable {
 		});
 
 		assessmentResults?.items.forEach((item) => {
-			const dbIndex = serverDatabases.indexOf(item.databaseName);
-			if (dbIndex === -1) {
-				serverLevelAssessments.push(item);
-			} else {
-				databaseLevelAssessments[dbIndex].issues.push(item);
+			if (item.appliesToMigrationTargetPlatform === 'AzureSqlManagedInstance') {
```

Comment:
Where are these platform names coming from? If there's a list of valid values then we should make this an enum type we can reference instead of just random strings

Reply:
This is coming from the assessments nuget package. I don't have a list all products they support. So for now I am using this hardcoded string.

Reply:
Could you at least make an enum with the values you know about? It'll help prevent issues and make it easier to add stuff in later. 

It looks like this is coming from the mssql extension so I'd suggest adding it to the typing file there.